plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/edit/struct/SQLTableManager.java

```diff
@@ -185,6 +185,12 @@ public DBEPersistAction[] getTableDDL(DBRProgressMonitor monitor, OBJECT_TYPE ta
 
         final DBERegistry editorsRegistry = DBWorkbench.getPlatform().getEditorsRegistry();
         SQLObjectEditor<DBSEntityAttribute, OBJECT_TYPE> tcm = getObjectEditor(editorsRegistry, DBSEntityAttribute.class);
+        /*
```

Comment:
It is not just for primary keys, it is for all constraints (primary, unique).
Maybe we should use more precise constraint types here consequently.